torch/utils/_benchmark/utils/timer.py

```diff
@@ -42,11 +44,13 @@ def __init__(
         globals.setdefault("torch", torch)
 
         self._stmt = stmt
+        self._setup = setup
         self._label = label
         self._sub_label = sub_label
         self._description = description
         self._env = env
         self._num_threads = num_threads
+        self._globals = globals
```

Comment:
This is not really related to this diff per se, but since you're already using static types in the argument list of `__init__`, you might as well use the syntax for directly declaring members to have types, by writing `_globals: Optional[dict]` inside the class scope. And then if you're going to do that, might as well make this a `dataclass` so you don't have to boilerplate all of the `__init__` stuff.

Reply:
Absolutely. I didn't know about dataclasses when I wrote this. I'll be sure to include that in the cleanup for 1.7.